### ISSUE TITLE
Security/ Reliability:  audit has_heartbeat

### ISSUE THREAD

**Issue by TheNotary** [EFForg/action-center-platform#26]
We're using a gem 'has_heartbeat' that enables an endbpoint, '/heartbeat' that checks the database connection and uptime of our system.  

Is there something logging stats on act.eff.org somewhere?  

Maybe this should be removed all-together, airbreak is a dependency of this gem which may being leaking log information that we wouldn't want it to (and has the potential to throw stack-level too deep which either masks the actual root of a critical error, or causes one).

**Comment by TheNotary** [comment]
We don't use the end point has_heartbeat exposes for monitoring, so we can eliminate this gem from the Gemfile at will.

**Comment by mfb** [comment]
I believe AWS itself is hitting the heartbeat endpoint? You can see it in the nginx logs.

10.200.166.126 - - [08/Oct/2015:18:02:48 -0700] "GET /heartbeat HTTP/1.1" 200 39 "-" "ELB-HealthChecker/1.0"
10.196.43.14 - - [08/Oct/2015:18:02:50 -0700] "GET /heartbeat HTTP/1.1" 200 39 "-" "ELB-HealthChecker/1.0"

**Comment by TheNotary** [comment]
Good catch.  Ben says it's the load balancer.

```
ben: [...]that's the load balancer
ben: [it's to] verify that it is operating correctly
ben: we might be able to change the url
ben: when multiple instances are load balanced, the balancer removes instances that are returning HTTP errors
ben: it is important for it to know if the instance has connectivity to it's DB backend
ben: so the url it hits should involve a DB transaction
ben: we can make it any url and port
```

ATM, we're running only one instance, but generally I guess it's a good idea to have an endpoint like this.  This has_heartbeat feels way more bulky than it needs to be though, I'd rather just add a controller in the code to test that, say `ActionPage.count > 0` and respond 200, else fail and respond 500.  The impact on the code base is 1 new line in `config/routes.rb` and ~4 new lines in app/controllers/robots_controller.rb and that's all that's needed.

**Comment by TheNotary** [comment]
This is perfect for you vbrown608.  

## tl;dr

- eliminate the has_heartbeat dependency
- setup the app to respond to the '/heartbeat' end point
- To prove that the app is connected to the database, just do a count of the action_pages table to see that it at least has 0 records

Start by telling me you'd prefer a TDD workflow and I'll help you with writing a failing test.
